### ISSUE TITLE
Failure to load observation file does not give descriptive message

### ISSUE THREAD

**Issue by mortalisk** [equinor/ert#4659]
As per https://wiki.equinor.com/wiki/index.php/Debugging error message for OBS_CONFIG is confusing It should indicate whether:

* the input file path exists or not.
* the input file format is incorrect

**Comment by oysteoh** [comment]
Probably resolved by #4661 - @andreas-el  do you verify?

**Comment by andreas-el** [comment]
The cases:

## Empty file:
```
ERT crashed unexpectedly with "Empty observations file: /private/andrli/project/ert/test-data/poly_example/observations1".
See logfile(s) for details:
   /private/andrli/project/ert/logs/ert-log-2023-01-18T1034.txt
```

## Missing file:
```
Error(s) in configuration file test-data/poly_example/poly.ert: ['Can not find entry observations2 in /private/andrli/project/ert/test-data/poly_example ']
```

## Invalid Content:
```
WARNING: Skipping unexpected token "skdfjklsafsdajkldfjas".
WARNING: Skipping unexpected token "sdgkdsgjklsdklfjga".
WARNING: Skipping unexpected token "dsgfakjgkldj".
```
:warning:  Not in suggester

## No OBS_CONFIG keyword
```
(testenv) andrli@be-lx919901 ~/project/ert (main)$ ert gui test-data/poly_example/poly.ert 
ERT crashed unexpectedly with "'FmuObs' object has no attribute 'getName'".
See logfile(s) for details:
   /private/andrli/project/ert/logs/ert-log-2023-01-18T1229.txt
```
```
2023-01-18 12:29:37,271 - ert.__main__ - MainThread - ERROR - ERT crashed unexpectedly with "'FmuObs' object has no attribute 'getName'"
Traceback (most recent call last):
  File "/private/andrli/project/ert/src/ert/__main__.py", line 564, in main
    args.func(args)
  File "/private/andrli/project/ert/src/ert/__main__.py", line 202, in run_gui_wrapper
    run_gui(args)
  File "/private/andrli/project/ert/src/ert/gui/gert_main.py", line 38, in run_gui
    window, ens_path = _start_initial_gui_window(args, log_handler)
  File "/private/andrli/project/ert/src/ert/gui/gert_main.py", line 111, in _start_initial_gui_window
    return _setup_main_window(ert, args, log_handler), res_config.ens_path
  File "/private/andrli/project/ert/src/ert/gui/gert_main.py", line 190, in _setup_main_window
    plugin_handler = PluginHandler(ert, ert.getWorkflowList().getPluginJobs(), window)
  File "/private/andrli/project/ert/src/ert/gui/tools/plugins/plugin_handler.py", line 18, in __init__
    plugin = Plugin(self.__ert, job)
  File "/private/andrli/project/ert/src/ert/gui/tools/plugins/plugin.py", line 22, in __init__
    self.__name = script.getName()
AttributeError: 'FmuObs' object has no attribute 'getName'
```

**Comment by oysteoh** [comment]
> No OBS_CONFIG keyword
This is related to an issue in subscript you looked at today @eivindjahren ?

**Comment by eivindjahren** [comment]
@oysteoh The `FmuObs' object has no attribute 'getName'` is related yes, and is now fixed by 63281cee36f93d1bb3fcd3a5ae916237c014f9a5
